Source/WebCore/dom/Element.cpp

```diff
@@ -1984,7 +2008,15 @@ ExceptionOr<void> Element::setAttribute(const AtomString& qualifiedName, const A
     auto caseAdjustedQualifiedName = shouldIgnoreAttributeCase(*this) ? qualifiedName.convertToASCIILowercase() : qualifiedName;
     unsigned index = elementData() ? elementData()->findAttributeIndexByName(caseAdjustedQualifiedName, false) : ElementData::attributeNotFound;
     auto name = index != ElementData::attributeNotFound ? attributeAt(index).name() : QualifiedName { nullAtom(), caseAdjustedQualifiedName, nullAtom() };
-    setAttributeInternal(index, name, value, InSynchronizationOfLazyAttribute::No);
+
+    auto sink = nullString();
```

Comment:
Worth adding a flag check here to skip all the trusted types related code if the flag is off. The code does handle it itself but this will be a hotpath so no harm in doing it early.

Reply:
It's handled at L1983, so not sure we need the flag check here.